DropDown/src/DropDown.swift

```diff
@@ -635,16 +657,13 @@ extension DropDown {
 	}
 	
 	private func fittingWidth() -> CGFloat {
-		struct Static {
-			static let templateCell = DropDownCell.Nib.instantiateWithOwner(nil, options: nil)[0] as! DropDownCell
-		}
-		
+		let templateCell = cellNib.instantiateWithOwner(nil, options: nil)[0] as! DropDownCell
```

Comment:
For performance issue, we could avoid to (re)create the `templateCell` every time we are calling `fittingWidth()`. Couldn't we use a `Dropdown` `private var` like so?
```swift
class DropDown: UIView {

	private var templateCell = DropDownCell?

	public var cellNib = UINib(nibName: "DropDownCell", bundle: NSBundle(forClass: DropDownCell.self)) {
		didSet {
			tableView.registerNib(cellNib, forCellReuseIdentifier: DPDConstant.ReusableIdentifier.DropDownCell)
			templateCell = cellNib.instantiateWithOwner(nil, options: nil)[0] as! DropDownCell
			reloadAllComponents()
		}
	}

	func setup() {
		// tableView.registerNib...
		templateCell = cellNib.instantiateWithOwner(nil, options: nil)[0] as! DropDownCell
		//...
	}

}
```